document/cloud-client/analyze_form.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def analyze_form(project_id, input_uri):
+    """Parse a form"""
+    # [START analyze_form]
+    from google.cloud import documentai_v1beta2 as documentai
+    from google.cloud.documentai_v1beta2 import types
+
+    client = documentai.DocumentUnderstandingServiceClient()
+
+    gcs_source = types.GcsSource(uri=input_uri)
```

Comment:
Under the one import all of these would switch to. 
```suggestion
    gcs_source = documentai.types.GcsSource(uri=input_uri)
```